napari_aicssegmentation/view/_util.py

```diff
@@ -0,0 +1,16 @@
+from PyQt5.QtWidgets import QComboBox
+
+from napari_aicssegmentation.widgets.form import FormRow
+
+
+def dropdown_row(label: str, placeholder: str, enabled=False) -> FormRow:
```

Comment:
I like that you're making this reusable - but I'm really not a fan of importing just methods like this if we can help it. It makes the code less readable because there is nothing obvious that points to the fact that this method comes from another file / module. Also, in general when we have single purpose directories like `/view` or `/controller`, I like to keep those clean and put utility files elsewhere.

Can we maybe turn this into a static class and stick in the `/util` folder that already exists? We can call it `FormUtils` or something along those lines.
```
class FormUtils:
     @staticmethod
     def dropdown_row(...):
```

Reply:
I know this is opinionated (it's a coding style comment more than anything) - but it'll be more consistent with what I've already put in place in the rest of the codebase.

Reply:
Thanks for the explanation! Learned something new. I made a class UiUtils and stuck it in the existing /util folder :)